Gemfile

```diff
@@ -1,40 +1,41 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
```

Comment:
please don't specify version of ruby in plain mode - use parsing of .ruby-version file here